lib/src/websocket/messages.dart

```diff
@@ -57,10 +57,10 @@ class InitOperation extends GraphQLSocketMessage {
     jsonMap['type'] = type;
 
     if (payload != null) {
-      jsonMap['payload'] = json.encode(payload);
+      jsonMap['payload'] = payload;
     }
 
-    return json.encode(jsonMap);
+    return jsonMap;
```

Comment:
This part should still be encoded, line 60 was cousing the issue I believe

Reply:
For my code, it is necessary to return the plain jsonMap instead of encoding it in order for the init payload to work properly.

Reply:
Wouldn't it be possible to add an unescaped method that doesn't do the encoding alongside the regular one?

And one more thing, you can't justify a pull by saying that it fixes your use case.